packages/crawler/src/report.ts

```diff
@@ -69,6 +69,13 @@ const lambdaHandler = async ({
     const name = `github/${gitHubRepositoryFullName}`;
     const revision = await git.revparse(["HEAD"]);
 
+    let report = require(`${jscpdReportLocalPath}/jscpd-report.json`);
```

Comment:
Please use `fs.readFileSync()` and `JSON.parse()`. 
CommonJS is a little legacy. 

https://nodejs.org/api/fs.html#fs_fs_readfilesync_path_options

Reply:
done